### PR TITLE
added the experimental release note for relative color syntax in firefox

### PR DESCRIPTION
**Pull request by dletorey**
### Description

Added the experimental release note for reative color syntax

### Motivation

Working on [Enable relative color syntax on nightly #33517](https://github.com/mdn/content/issues/33517)

### Related issues and pull requests

- [BCD PR](https://github.com/mdn/browser-compat-data/pull/23340)

### CODE REVIEW

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 1]
```diff
@@ -275,6 +275,46 @@ See [Firefox bug 1889133](https://bugzil.la/1889133) for more details.
   </tbody>
 </table>
 
+### Relative colors syntax
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:278]: Minor one:

```suggestion
### Relative color syntax
```

File: files/en-us/mozilla/firefox/experimental_features/index.md [hunk 2]
```diff
@@ -275,6 +275,46 @@ See [Firefox bug 1889133](https://bugzil.la/1889133) for more details.
...
+
+The [relative color syntax](/en-US/docs/Web/CSS/CSS_colors/Relative_colors) is now enabled by default in Nightly and when the `layout,css.relative-color-syntax.enabled` preference is set to `true` in other versions. This allows the creation of colors from other colors using the [`color()`](/en-US/docs/Web/CSS/color_value/color) functional notation and other [color spaces](/en-US/docs/Glossary/Color_space) such as [`lch()`](/en-US/docs/Web/CSS/color_value/lch), [`lab()`](/en-US/docs/Web/CSS/color_value/lab), etc. [Firefox bug 1701488](https://bugzil.la/1701488).
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/experimental_features/index.md:280]: What do you think about this?

```suggestion
[Relative color syntax](/en-US/docs/Web/CSS/CSS_colors/Relative_colors) is now enabled by default in the Nightly release and via the `layout.css.relative-color-syntax.enabled` preference set to `true` in other release versions. Relative color syntax allows you to create a color value relative to an origin color, and can allow you to change a color in a different [color space](/en-US/docs/Glossary/Color_space) using [color functions](/en-US/docs/Web/CSS/CSS_colors#functions) ([Firefox bug 1701488](https://bugzil.la/1701488)).
```

File: files/en-us/mozilla/firefox/experimental_features/index.md (hunk 1 above)
Comment (dipikabh) [reply files/en-us/mozilla/firefox/experimental_features/index.md]: or "Relative colors"?

**Review by bsmth — APPROVED** [review]
Super, thank you!

**Pull request merged**